api/src/controllers/search.js

```diff
@@ -42,6 +43,8 @@ let generateExpArray = async function (field, logicalOperator = '$or', compariso
           return convertValue(element);
         });
         return { [logicalOperator]: [{ [item]: { $in: arrayExp } }] };
+      } else if (!Array.isArray(entry) && comparisonOperator === '$in') {
```

Comment:
I searched and couldn't find the `_in` value being used anywhere else, so I don't think making this change will cause any issues.